ecl/network/v2/qos_options/testing/fixtures.go

```diff
@@ -0,0 +1,95 @@
+package testing
+
+import "github.com/nttcom/eclcloud/ecl/network/v2/qos_options"
+
+const ListResponse = `
```

Comment:
I think only one service ID(e.g. aws_service_id) is not blank and the other IDs are blank. I mean two or more IDs never are not blank at the same time. Can you confirm actual api response?